src/encoded/schemas/hic_quality_metric.json

```
@@ -17,127 +17,286 @@
+                            "2_alignment_duplicates",
+                            "2_alignment_unique",
+                            "0_alignments",
+                            "pct_0_alignments",
+                            "pct_1_alignment_unique",
+                            "pct_1_alignment_duplicates",
+                            "pct_sequenced_1_alignment_unique",
+                            "pct_sequenced_1_alignment_duplicates",
+                            "pct_sequenced_2_alignment_unique",
+                            "pct_2_alignment_duplicates",
+                            "pct_2_alignment_unique",
+                            "pct_1_alignment",
+                            "sequenced_reads",
+                            "1_alignment",
+                            "1_alignment_unique",
+                            "1_alignment_duplicates"
+                        ]
+                    },
+                    "properties": {
+                        "run_type": {
+                            "enum": [
+                                "paired-ended"
+                            ]
+                        }
+                    }
+                }
+            ]
+        }
+    },
     "properties": {
         "schema_version": {
-            "default": "1"
+            "default": "2"
+        },
+        "run_type": {
+            "title": "Run type of sequencing run",
+            "type": "string",
+            "enum": [
+                "single-ended",
+                "paired-ended"
+            ]
+        },
+        "sequenced_reads": {
+            "title": "Number of sequenced read",
+            "type": "integer",
+            "minimum": 0
         },
         "sequenced_read_pairs": {
             "title": "Number of sequenced read pairs",
             "type": "integer",
             "minimum": 0
         },
-        "normal_paired": {
-            "title": "Number of normal paired",
+        "no_chimera_found": {
+            "title": "Number of no chimera found",
+            "type": "integer",
+            "minimum": 0
+        },
+        "0_alignments": {
+            "title": "Number of 0 alignments",
+            "type": "integer",
+            "minimum": 0
+        },
+        "1_alignment": {
+            "title": "Number of 1 alignment",
+            "type": "integer",
+            "minimum": 0
+        },
+        "1_alignment_unique": {
+            "title": "Number of 1 alignment unique",
+            "type": "integer",
+            "minimum": 0
+        },
+        "1_alignment_duplicates": {
+            "title": "Number of 1 alignment duplicates",
+            "type": "integer",
+            "minimum": 0
+        },
+        "2_alignments": {
+            "title": "Number of 2 alignments",
+            "type": "integer",
+            "minimum": 0
+        },
+        "2_alignments_a_b": {
+            "title": "Number of 2 alignments a_b",
```

Same as below, I think using the Juicer description would be good here:
```suggestion
            "title": "Number of 2 alignments (A...B)",
```

---

ok